tools/Dockerfile

```diff
@@ -41,9 +41,9 @@ RUN pacman --sync --refresh --noconfirm \
     terraform \
     yamllint
 
-# TODO better way to install k3d?
-RUN curl -L https://github.com/k3d-io/k3d/releases/download/v5.4.1/k3d-linux-amd64 > /usr/local/bin/k3d \
-    && chmod +x /usr/local/bin/k3d
+ARG K3D_PATH=/usr/local/bin/k3d
+RUN curl -L https://github.com/k3d-io/k3d/releases/latest/download/k3d-linux-amd64 -o $K3D_PATH -s \
```

Comment:
I would prefer using the current `/usr/local/bin/k3d` instead of another `ARG` (unneeded layer)